git_machete/client.py

```diff
@@ -1851,8 +1851,7 @@ def checkout_github_prs(self,
         remote, (org, repo) = self.__derive_remote_and_github_org_and_repo()
         current_user: Optional[str] = git_machete.github.derive_current_user_login()
         if not current_user and my_opened_prs:
-            raise MacheteException(
-                "Could not determine current user name, please check your token.")
+            warn("Could not determine current user name, please check your token.")
```

Comment:
I'd say this should be a non-recoverable failure when invoked from `git machete github checkout-prs --mine` (since it is supposed to do just one thing... and it cannot), but only a warning when invoked from `git machete github sync` or `git machete clean -H` (since other parts of the flow can still be executed regardless).

Reply:
Also... in case when just a warning is raised, shouldn't we `return` from the method? rather than continue (now fruitless) flow